src/components/HomePage/PlayRadioButton.tsx

```diff
@@ -65,14 +89,8 @@ const PlayRadioButton = () => {
             {t('play-radio')}
           </Button>
         )}
-
-        {shouldShowStationName && (
-          <div className={styles.stationInfo}>
-            <span className={styles.stationTitle}>{stationInfo.title}</span>{' '}
-            <Link href="/radio" className={styles.editStationButton}>
-              ({t('change')})
-            </Link>
-          </div>
+        {audioService.getSnapshot().context.radioActor && (
+          <RadioInformation radioActor={audioService.getSnapshot().context.radioActor} />
```

Comment:
`audioService.getSnapshot().context.radioActor` is being repeated twice, we should put it in a variable